vrtool_docs/Installaties/VRUtils.md

```diff
@@ -37,10 +37,6 @@ poetry install
 ```
 
 #### Let op
-- Als er zich een probleem voordoet bij Stap 5 met de package ```peewee```, installeer deze dan handmatig door de volgende commandline in te typen: ```pip install peewee==3.16.2```. Doe daarna nog eens: ```poetry install```
-
 - Bij Stap 5 lukt de installatie soms niet ineens, probeer het dan nogmaals (het kan zijn dat je het 3 keer moet proberen): ```poetry install```
-Je kunt zien dat de installatie niet gelukt is als er in de command line 'PoetryException' in rood geschreven staat: 
-![Poetry_error_circled.PNG](Poetry_error_circled.PNG)
-
+- Als er zich een probleem voordoet bij Stap 5 met de package ```peewee```, installeer deze dan handmatig door de volgende commandline in te typen: ```pip install peewee==3.16.2```. Doe daarna nog eens: ```poetry install```
```

Comment:
KEEP: This is a screenshot of the error you get when poetry is not properly installed. I think a handy visual add that we can leave.